releasenotes/notes/33145.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
```

Comment:
I think it is fine to add this without a release notes.

Reply:
Alright, thanks!